bims/views/non_validated_user_list.py

```diff
@@ -0,0 +1,52 @@
+# coding=utf-8
+from braces.views import LoginRequiredMixin
+from django.views.generic import ListView
+from bims.models.biological_collection_record import BiologicalCollectionRecord
+
+
+class NonValidatedObjectsUserView(LoginRequiredMixin, ListView):
+
+    model = BiologicalCollectionRecord
```

Comment:
@ann26 
What happens if user adds a data in the FishCollectionRecord and want to edit specific field from that model? Fish model has habitat field and BioCollectionRecord doesn't.

Reply:
that will not show up